scripts/linter_utils_test.py

```diff
@@ -27,6 +27,16 @@
 class MemoizeTest(test_utils.GenericTestBase):
     """Tests for the memoize function."""
 
+    def test_memoize_always_returns_value(self):
+        @linter_utils.memoize
+        def return_one():
+            """Returns the integer 1."""
+            return 1
+
+        self.assertEqual(return_one(), 1)
+        self.assertEqual(return_one(), 1)
```

Comment:
A more powerful test would change what return_one() returns after this call, and the memoized function should still return 1.